src/robot/libraries/BuiltIn.py

```diff
@@ -613,9 +613,16 @@ def should_be_equal(self, first, second, msg=None, values=True):
         for example, string ``false`` or ``no values``. See `Boolean arguments`
         section for more details.
 
+        ``ignore_case`` is False by default.  It is a boolean value, and if True
+        indicates that 'first' and 'second' should be compared case-insensitively,
+        provided that 'first' and 'second' are string types.
```

Comment:
- Docs should refer to `Boolean arguments` section about possible values for `ignore_case`. You can check how other keywords here do it. Values should also be handled according to that section, but more about that below.
- Should use double-backticks style also with arguments `first` and `second`.
- Should add a note that this is a new feature in Robot Framework 3.0.1.